examples/grpc/CMakeLists.txt

```diff
@@ -39,12 +39,7 @@ endif()
 foreach(_target client server)
   add_executable(${_target} "${_target}.cpp")
   target_link_libraries(
-    ${_target}
-    example_grpc_proto
-    protobuf::libprotobuf
-    gRPC::grpc++
-    gRPC::grpc++_reflection
```

Comment:
Why is `grpc++_reflection` removed?

Reply:
Because there is no such package / target / library, at least not in `vcpkg` build of gRPC. Can you point me where this target is declared?

Reply:
I might be mistaken on this one. But it all compiles without this target. I see where it is mentioned in gRPC build. But I had an issue with that one on Windows.. we do build all examples across all CI loops for CMake?

Reply:
It is a library target as below. I think we build all examples in our CMake CI. We probably don't use use the reflection function in gRPC.

https://github.com/grpc/grpc/blob/master/CMakeLists.txt#L3146

Reply:
Yes, all examples are built so hopefully should be fine :) Though I faintly remember getting link errors without this library, but all good if it builds fine in CI.